aspnetcore/performance/rate-limit.md

```diff
@@ -141,7 +145,7 @@ app.UseEndpoints(endpoints =>
 
 #### Apply rate limiting to Blazor Server Pages
 
-To set rate limiting to all pages,  `RequireRateLimiting(Policy)` can be specified on the MapRazorComponents call, for example:
+To set rate limiting to all pages,  [`RequireRateLimiting(Policy)`](/dotnet/api/microsoft.aspnetcore.builder.ratelimiterendpointconventionbuilderextensions.requireratelimiting) can be specified on the MapRazorComponents call, for example:
```

Comment:
FYI @samsp-msft ... We have a crappy, not finished, but working tool to help with these XREF cross-links at ...

https://dotnet.github.io/blazor-samples/